New-work.html

```diff
@@ -17,8 +17,8 @@
 
 
 <body class="r">
-      <div class="d">  <div class="h1"><h1>BLOG</h1></div>
-        <div class="h2"><h2>News on covid 19</h2></div>
+      <div class="d">  <div class="h1"><h1>Gitting</h1></div>
```

Comment:
Wrong class nomenclature for css element

```suggestion
      <div class="icon-title">  <div class="h1"><h1>Gitting</h1></div>
```